docs/0.4/index.md

```diff
@@ -18,19 +18,19 @@ application interfaces, and transaction processing.
 
 This topic describes Splinter's key benefits:
 
-* [True privacy with circuits](#bookmark-private-circuits)
-* [Distributed ledger platform as a service](#bookmark-dl-service)
-* [Flexibility with Splinter's modular architecture](#bookmark-arch)
-* [Smart contracts that can share state](#bookmark-sm-share-state)
-* [Quick startup for application development](#bookmark-dev-startup)
-* [Event subscription for state changes](#bookmark-event-sub)
-* [Complex patterns for data flow](#bookmark-complex-data-flow)
-* [Auditable open source](#bookmark-open-source)
+* [True privacy with circuits](#true-privacy-with-circuits)
+* [Distributed ledger platform as a service](#distributed-ledger-platform-as-a-service)
+* [Flexibility with Splinter's modular architecture](#flexibility-with-splinters-modular-architecture)
+* [Smart contracts that can share state](#smart-contracts-that-can-share-state)
+* [Quick startup for application development](#quick-startup-for-application-development)
+* [Event subscription for state changes](#event-subscription-for-state-changes)
+* [Complex patterns for data flow](#complex-patterns-for-data-flow)
+* [Auditable open source](#auditable-open-source)
 
 You can also learn about [when Splinter might not be the right
 choice](#bookmark-why-not-splinter).
```

Comment:
Thanks for doing this!  This anchor tag got missed.

Reply:
Good catch! Fixed.